run-python3.js

```diff
@@ -7,7 +7,7 @@
 // Invoke it like so: "node run-python3.js script.py"
 
 const cp = require("child_process");
-const extension = require("@azure-tools/extension");
+const extension = require("@autorest/system-requirements");
 
 async function runPython3(scriptName, debug = "") {
   const command = ["python", scriptName, debug];
```

Comment:
If the CI works fine with this, Can you try replacing this with 
```
const command = await extension.patchPythonPath(["python", scriptName, debug]);
```

and remove the updatePythonPath below that way we can remove that legacy method.

Reply:
Given the API:
```
export declare const patchPythonPath: (command: PythonCommandLine, requirement: SystemRequirement) => Promise<string[]>;
```

what should I use for `requirement`?

Reply:
I see this, should I use that?
```
const updatePythonPath = async (command) => {
    const newCommand = await (0, exports.patchPythonPath)(command, { version: ">=3.6", environmentVariable: "AUTOREST_PYTHON_EXE" });
    command[0] = newCommand[0];
    return newCommand;
};
```

Reply:
Yeah, or whatever is the current requirement for the python SDK. That way you are in control of this